uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/Role/DrupalRoleService.php

```diff
@@ -21,19 +15,39 @@ public function getRoleNames() {
    * @inheritdoc
    */
   public function getLatestLoginByRole($roleName) {
```

Comment:
`getLatestLoginForRole` seems to be a better name for this function. `getLatestLoginByRole` gives the impression that the function is retrieving latest login per role.